Dockerfile

```diff
@@ -60,7 +60,7 @@ RUN apk add --virtual .build-deps \
 
 COPY . /opt/mautrix-telegram
 RUN apk add git && pip3 install .[speedups,hq_thumbnails,metrics,e2be] \
-  && pip3 install 'git+https://github.com/vector-im/mautrix-python@bump-conn-pool-limit#egg=mautrix' \
+  && pip3 install 'git+https://github.com/vector-im/mautrix-python@master#egg=mautrix' \
```

Comment:
Is there no tag to pin to instead of using `master` which means you might pull new code on every build, making each build different?

Reply:
There is a tag now and the builds should be reproducible(ish).